frontend/js/src/common/brainzplayer/ProgressBar.tsx

```diff
@@ -25,9 +25,7 @@ const TOOLTIP_TOP_OFFSET: number = 102;
 function ProgressBar(props: ProgressBarProps) {
   const { durationMs, progressMs, seekToPositionMs } = props;
   const [tipContent, setTipContent] = React.useState(TOOLTIP_INITIAL_CONTENT);
-  const progressPercentage = Math.round(
-    Number((progressMs * 100) / durationMs)
-  );
+  const progressPercentage = Math.round((Number(progressMs * 100) / durationMs) * 100) / 100;
```

Comment:
That seems a little bit of a roundabout way to round up with more precision; try this :)

```suggestion
  const progressPercentage = Number(progressMs * 100 / durationMs).toFixed(2);
```